src/hooks/useVaultProperties.ts

```diff
@@ -0,0 +1,26 @@
+import { EvmAddress } from "@/types/types";
+import { useReadHederaVaultAsset } from "@/hooks/eip4626/useReadHederaVaultAsset";
+import { useReadHederaVaultShare } from "@/hooks/eip4626/useReadHederaVaultShare";
+import { useReadHederaVaultUserContribution } from "@/hooks/eip4626/useReadHederaVaultUserContribution";
+import { useReadBalanceOf } from "@/hooks/useReadBalanceOf";
+
+export const useVaultProperties = (vaultAddress: EvmAddress) => {
+  const { data: vaultAssetAddress } = useReadHederaVaultAsset(vaultAddress);
+  const { data: vaultAssetUserBalance } = useReadBalanceOf(
+    vaultAssetAddress as EvmAddress,
+  );
+  const { data: vaultShareAddress } = useReadHederaVaultShare(vaultAddress);
+  const { data: shareUserBalance } = useReadBalanceOf(
+    vaultShareAddress as EvmAddress,
+  );
+  const { data: userContribution } =
+    useReadHederaVaultUserContribution(vaultAddress);
+
+  return {
+    vaultAssetAddress,
+    vaultShareAddress,
+    vaultAssetUserBalance,
+    shareUserBalance,
+    userContribution,
```

Comment:
There are downsides of having them together, considering every time we use this hook, all of those things will run, even if we need only some of them in the component.

Also current structure tempts to keep stuffing this generic `src/hooks/useVaultProperties.ts` hook with more and more things.

We already have all these queries as separate hooks, lets use them directly in the components as is, as it brings some benefits like ability to check for errors and request statuses.
I recommend dropping this hook completely.